packages/engine/tests/units/globals/access_from_init/src/behaviors/test.js

```diff
@@ -1,4 +1,4 @@
 /**
- * Empty behavior as we only test setting globals from `init.js`
+ * Empty behavior as we only test accessing globals from the initial state
```

Comment:
Do we actually need any behaviors?

Reply:
It's needed to store the behavior key

Reply:
Also, at least JS is emitting a warning if no behavior is found.